app/src/main/java/com/example/party_player/MainActivity.kt

```diff
@@ -116,7 +114,7 @@ class MainActivity : AppCompatActivity() {
         })
     }
 
-    fun previousSong() {
+    private fun previousSong() {
```

Comment:
I changed this function to private for more clarification.

Reference: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules